cypress/integration/report_page_test.js

```diff
@@ -0,0 +1,104 @@
+describe("Report Page Test", function(){
+    beforeEach(function(){
+        cy.searchKeyword("GST");
+        cy.get(".result").click();
+        cy.url().should("include","/report");
+    });
+
+    it("can filter the topic of a search result", function(){
+        cy.get("main").find("input").type("Publications{enter}",{delay:100});
+        cy.get('.text-highlight').should("include.text","Publications").click();
+        cy.get('.report-item-title').should("include.text","Publications");
+        cy.get('.im-table').should("exist");
+        cy.isInViewport('.im-table');
+    });
+
+    it("can access a topic from the left sidebar", function(){
+        cy.get(".toc").within(() => {
+            cy.contains("Data").click();
+            cy.contains("Exons").click();
+        })
+
+        cy.get('.report-item-title').contains("Exons")
+        .parent().parent(".report-item")
+        .as('exonsTable');
+        cy.wait(100); 
+        cy.isInViewport("@exonsTable");
+    });
+
+    it("can collapse and expand a section on report page", function(){
+        cy.get('.report-table-heading').contains("Visualizations")
+        .parent(".report-table")
+        .as('visualizationsSection');
+
+        cy.get("@visualizationsSection").within(() => {
+            cy.get(".report-item").should("exist");
+            cy.get(".icon-chevron-up").click();
+            cy.get(".report-item").should("not.exist");
+        })
+    });
```

Comment:
Can you make this use the *Data* heading instead? The visualizations may change in the future, and the header will disappear if there aren't any.